metadata-ingestion/src/datahub/telemetry/telemetry.py

```diff
@@ -123,14 +123,17 @@ def init_tracking(self) -> None:
             return
 
         logger.info("Sending init Telemetry")
-        self.mp.people_set(
-            self.client_id,
-            {
-                "datahub_version": datahub_package.nice_version_name(),
-                "os": platform.system(),
-                "python_version": platform.python_version(),
-            },
-        )
+        try:
+            self.mp.people_set(
+                self.client_id,
+                {
+                    "datahub_version": datahub_package.nice_version_name(),
+                    "os": platform.system(),
+                    "python_version": platform.python_version(),
+                },
+            )
+        except Exception as e:
+            logger.debug(f"Error reporting telemetry: {e}")
```

Comment:
Shouldn't this should be at warning level?

Reply:
fixed